src/utils/bivariate/types/stat.types.ts

```diff
@@ -19,10 +19,16 @@ export type Step = {
 
 /* Divisor and denominator pair */
 export type Quotient = [string, string];
+export type Quotients = {
```

Comment:
Very similar names. What both of them means? I think their names should give move info or some comments should be placed around

Reply:
hm, I see. 
This is because it's added in response on backend side
https://kontur.fibery.io/Tasks/Task/Add-info-about-indicator-sentiments-and-labels-for-presets-on-DN2-be-10722
So now we have quotient  and quotients. Types describes what's inside, so maybe put a comment here?
Or maybe change name to QuotientDetails ?

Reply:
One of those, don't know which one's better